pkg/plugins/builtin/script/script.go

```diff
@@ -0,0 +1,108 @@
+package script
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	gexec "os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/ovh/utask/pkg/plugins/taskplugin"
+)
+
+// the script plugin execute scripts
+var (
+	Plugin = taskplugin.New("script", "0.1", exec,
+		taskplugin.WithConfig(validConfig, Config{}),
+	)
+)
+
+// Config is the configuration needed to execute a script
+type Config struct {
+	File    string   `json:"file,required"`
+	Argv    []string `json:"argv,omitempty"`
+	Timeout string   `json:"timeout_seconds,omitempty"`
+}
+
+func validConfig(config interface{}) error {
+	cfg := config.(*Config)
+
+	if cfg.File == "" {
+		return errors.New("file is missing")
+	}
+
+	if cfg.Timeout != "" {
+		if _, err := strconv.ParseUint(cfg.Timeout, 10, 64); err != nil {
```

Comment:
keep in mind that this doesnt allow templating for this field, since {{.config.whatever}} wont pass ParseUint.
Probably acceptable in this specific case.

Reply:
Why? from the moment we receive a correct string who contains a number > 0 it will works.

Reply:
validConfig is called without any templated values substitution